File: ed1/gara2/pesci/att/pesci.pas
function mangia(N: longint; var direzione, dimensione: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    mangia := 42;
end;


const
    MAXN = 100000;
var
    N, i                  : longint;
    direzione, dimensione : array[0..MAXN-1] of longint;
    fr, fw                : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        readln(fr, direzione[i], dimensione[i]);

    writeln(fw, mangia(N, direzione, dimensione));
    close(fr);
    close(fw);
end.

File: ed1/gara2/pesci/att/pesci.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int mangia(int N, int direzione[], int dimensione[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int direzione[MAXN], dimensione[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(2 == fscanf(fr, "%d%d", &direzione[i], &dimensione[i]));

    fprintf(fw, "%d\n", mangia(N, direzione, dimensione));
    fclose(fr);
    fclose(fw);
    return 0;
}
